Users/Install.rst

```diff
@@ -90,13 +90,20 @@ Virtualenv is probably what you want to use during development.
 You can read more about
 it at the `virtualenv documentation <http://virtualenv.readthedocs.org>`_.
 
-First, we need to install virtualenv to the system. You may already have this
-installed as ``virtualenv`` or ``pyvenv``. If you do not, this can be done
-with ``pip3`` easily:
+First, we need to install virtualenv to the system. You may already
+have this installed as ``virtualenv`` or ``pyvenv``. You can
+check if it's already installed by typing:
 
 ::
 
-    $ pip3 install virtualenv
+    $ virtualenv --version
+
+If you see a version number, its already installed. If you do
+not, this can be done with pip3 easily:
+
```

Comment:
You can improve this by mentioning the output that the user will see if ``virtualenv`` is not installed.

Reply:
I have already mentioned that the user will not see a version number if `virtualenv` is not installed and I think the output message would be pretty obvious to know that its not installed yet